hwdb/70-libratbag-mouse.hwdb

```diff
@@ -69,31 +69,39 @@ mouse:usb:v046dpc041:name:*:
  RATBAG_HIDPP10_DPI_LIST=400;800;1600;2000
 
 # G5 2007
+# http://support.logitech.com/en_us/product/g5-laser-mouse-product/specs
 mouse:usb:v046dpc049:name:*:
  RATBAG_DRIVER=hidpp10
  RATBAG_HIDPP10_DPI_LIST=400;800;1600;2000
```

Comment:
IIRC, they are not unknown, they are just using the default value 1.
I guess we can still override this here.